src/plugins/speech/structure_plugin.js

```diff
@@ -94,6 +147,7 @@ define( ['structure_custom' ], function( VisuDesign_Custom ) {
         selectedVoice = defaultVoice;
       }
       utterThis.voice = selectedVoice;
+      console.log("saying '%s' in voice %s", text, selectedVoice.name);
```

Comment:
Are these `console.log` messages here on purpose (e.g. during the development and stabilizing phase) or were they forgotten to remove?